#!/usr/bin/python3
def raise_exception_msg(message=""):
    try:
        if name:
            pass
    except:
        raise NameError(message)
